### PR TITLE
The name is a misnomer - really just correcting meaning of L

### LINKED ISSUE #36: particle "length" interpretation is wrong
Right now the implementation of the "length" of the particle needs to be generalized to three flavors. Right now it only works for two. In the case where there are initially no heavy lepton neutrinos it doesn't matter (the evolution equations are the same), but should be fixed for the future.

### PR DESCRIPTION
**Pull request by srichers**
I spent some time trying to reconfigure the variables to evolve only the traceless part of the variables, but there were enough differences (and bugs) that I just decided to go with the simplest possible fix for the current incorrectness. Closes #36

### CODE REVIEW

File: Scripts/symbolic_hermitians/HermitianUtils.py [hunk 1]
```diff
@@ -93,6 +93,12 @@ def SU_vector_magnitude(self):
 
         return sympy.sqrt(mag2)
```
Comment (srichers) [inline Scripts/symbolic_hermitians/HermitianUtils.py:95]: The following is unused. It was a byproduct of my other tinkering, but I didn't see a reason to delete it.

File: Scripts/symbolic_hermitians/generate_code.py [hunk 2]
```diff
@@ -159,7 +159,7 @@ def delete_generated_files():
     deposit_vars = ["N","Fx","Fy","Fz"]
```
Comment (srichers) [inline Scripts/symbolic_hermitians/generate_code.py:159]: This essentially reverts the EOMs to what they were before the whole L business. Now L is no longer used in the EOMs but rather is just recorded as a constraint for the particle evolution.

File: Scripts/symbolic_hermitians/generate_code.py [hunk 3]
```diff
@@ -392,3 +392,12 @@ def sgn(t,var):
     # Write code to output file, using a template if one is provided
     # write_code(code, "code.cpp", args.output_template)
```
Comment (srichers) [inline Scripts/symbolic_hermitians/generate_code.py:394]: No not have L be in input. Rather, L is calculated from the input values of f.

**Review by dwillcox — APPROVED** [review]
Makes sense!

**Pull request merged**